expression/builtin_math.go

```diff
@@ -185,3 +185,13 @@ func builtinRound(args []types.Datum, ctx context.Context) (d types.Datum, err e
 	d.SetFloat64(types.Round(x, dec))
 	return d, nil
 }
+
+// See http://dev.mysql.com/doc/refman/5.7/en/mathematical-functions.html#function_conv
+func builtinConv(args []types.Datum, ctx context.Context) (d types.Datum, err error) {
```

Comment:
Please add TODO here and return unimplement error.
I think it is OK to remove the two functions. You only need to make parser happy.